rep-0153.rst

```diff
@@ -73,6 +78,9 @@ Index file
       For use case *1.* the values ``ros1`` and ``ros2`` will be used to
       distinguish the major ROS version.
 
+    * python_version: an optional integer describing the major version of
+      Python of the ROS distribution.
```

Comment:
Given that this is optional, should the REP define a default?

Reply:
I don't think there is any default value which would be reasonable / not have common cases where it would be wrong. The official `rosdistro` database is being update to define the value for every ROS distro: ros/rosdistro#22484.

Reply:
Well, given that the REP specifies that unknown keys are to be ignored, along with the fact that until very recently everything was python 2, clients will be assuming python 2 unless support for this key is added, right? In other words, it seems like the de-facto default is 2, here, but only because of assumptions made by consumers of the REP. That seems potentially problematic, but I suppose also mostly academic.

Reply:
> along with the fact that until very recently everything was python 2, clients will be assuming python 2 unless support for this key is added, right?

That is incorrect. While all ROS 1 distros up to Melodic are using Python 2 all ROS 2 distributions are using Python 3.

Reply:
Great point-- that distinction is currently hard-coded then, yes? Which comes back to this PR. Very good, thank you.